mw-webapp/src/component/table/columns.tsx

```diff
@@ -11,121 +15,129 @@ const DEFAULT_SUMMARY_TIME = 0;
 
 const columnHelper = createColumnHelper<DayReport>();
 
-const getObjectArrayItem = (arrayItem: JobDone | PlanForNextPeriod | CurrentProblem, getFullItem?: string) => {
-  return (
-    (JSON.stringify(arrayItem) === "{}") ?
-      <div />
-      :
-      <div key={arrayItem.uuid}>
-        {getFullItem}
-      </div>
+const getObjectArrayItem = (
+  arrayItem: JobDone | PlanForNextPeriod | CurrentProblem,
+  getFullItem?: string,
+) => {
+  return JSON.stringify(arrayItem) === "{}" ? (
+    <div />
+  ) : (
+    <div key={arrayItem.uuid}>
+      {getFullItem}
+    </div>
   );
 };
 
 const getStringArrayItem = (arrayItem: string, index: string) => {
-  return (
-    (!arrayItem) ?
-      <div />
-      :
-    //TODO: task #65 use flag instead of first index
-      <div key={index}>
-        <div className={arrayItem[INDEX_OF_CHECK_MARK] === "✓" ? styles.completed : styles.notCompleted}>
-          {arrayItem}
-        </div>
+  return !arrayItem ? (
```

Comment:
Why these changes? Why you deleted "(" . Explain please

Reply:
Must have been done automatically by prettier